modules/simple-payments/simple-payments.php

```diff
@@ -86,23 +88,26 @@ function parse_shortcode( $attrs, $content = false ) {
 
 	function output_shortcode( $data ) {
 		$items = '';
+		$cssPrefix = self::$css_classname_prefix;
+
 		if ( $data['multiple'] ) {
-		       $items="<div class='jetpack-simple-payments-items'>
-		       <input class='jetpack-simple-payments-items-number' type='number' min='1' value='1' id='{$data['dom_id']}_number'>
-		       </div>";
+			$items="<div class='${cssPrefix}-items'>
+				<input class='${cssPrefix}-items-number' type='number' value='1' id='{$data['dom_id']}_number' />
+			</div>";
 		}
-		$output = "
-<div class='{$data[ 'class' ]} jetpack-simple-payments__wrapper'>
-	<p class='jetpack-simple-payments__purchase-message'>
-	</p>
-	<div class='jetpack-simple-payments__title'>{$data['title']}</div>
-	<div class='jetpack-simple-payments__description'>{$data['description']}</div>
-	<div class='jetpack-simple-payments__price'>{$data['price']}</div>
-	{$items}
-	<div class='jetpack-simple-payments__button' id='{$data['dom_id']}_button'></div>
+
+		return "
+<div class='{$data['class']} ${cssPrefix}-wrapper'>
```

Comment:
I think adding the class name prefix as a variable is not very helpful. It's harder to read the styling then and I've never seen class naming in a variable. :) We are also not going to change the class name prefix I think -- why would we?